### PR TITLE
Fixes "Never show me this message" for certain analysis

### PR DESCRIPTION
**Pull request by javierarce**
This PR fixes an issue which caused the onboarding window for analysis to appear when users checked the "Never show me this message." box. 

The problem was that certain analyses share a common "generic type" and the analysis onboarding launcher wasn't using that generic type, but the analysis type (which is different for every kind of analysis).

Closes: #9284 

CR: @xavijam

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/components/onboardings/analysis/analysis-launcher.js [hunk 1]
```diff
@@ -37,7 +37,9 @@ var launcher = (function () {
     },
 
     launch: function (type, model) {
-      if (!LocalStorage.get(type) && ANALYSES_TYPES[type]) {
+      var genericType = ANALYSES_TYPES[type] && ANALYSES_TYPES[type].genericType ? ANALYSES_TYPES[type].genericType : type;
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/onboardings/analysis/analysis-launcher.js]: ANALYSES_TYPES[type] && ANALYSES_TYPES[type].genericType || type?

**Comment by xavijam** [comment]
Worth it a test?

**Comment by javierarce** [comment]
I don't think so, that would be throwing out money.

**Comment by xavijam** [comment]
:es:

**Pull request merged**